src/dev/lcd_hd44780.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+#ifndef DSY_LCD_HD44780_H
+#define DSY_LCD_HD44780_H
+
+#include "daisy_core.h"
+#include "per/gpio.h"
+
+
+class LcdHD44780
+{
+  public:
+    LcdHD44780() {}
+    ~LcdHD44780() {}
+
+    struct Config
+    {
+        bool         cursor_on;
+        bool         cursor_blink;
+        dsy_gpio_pin rs, en, d4, d5, d6, d7;
+    };
+
+    void Init(const Config &);
+    void Print(const char *);
+    void PrintInt(int);
+    void SetCursor(uint8_t, uint8_t);
+    void Clear();
```

Comment:
Add argument  names for and some (simple) doxygen reference documentation. 

for example:

```C++
/** Initializes the HDD44780 LCD display with the specified configuration */
void Init(const Config& cfg);
```

While the init function is fairly obvious, the others may not be as clear to the user. For example, I _assume_  `SetCursors` first argument is the x position, and the second argument is the y position, but I can't be sure without going all the way to the cpp file.

Reply:
The SetCursor() is defined as SetCursor(iint8_t row, uint8_t col) so the first arg is row, the second is col.

I am sorry but I am very new to GitHub so I am not really sure what you want me to change or do! :)